code/modules/mob/living/simple_animal/abnormality/teth/my_sweet_home.dm

```diff
@@ -0,0 +1,87 @@
+//Brain go brrrr.
+/mob/living/simple_animal/hostile/abnormality/my_sweet_home
+    name = "My Sweet Home"
+    desc = "This cozy little house is a safe nest built only for you. Everything is here for you..."
+    icon = null
+    icon_state = null
+    icon_living = null
+    icon_dead = null
+    var/icon_aggro = null
+    threat_level = TETH_LEVEL
+    can_breach = TRUE
+    del_on_death = TRUE
+    maxHealth = 600
+    health = 500
+    move_to_delay = 4
+    damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0.9, WHITE_DAMAGE = 0.9, BLACK_DAMAGE = 1.2, PALE_DAMAGE = 2)
+    melee_damage_lower = 6
+    melee_damage_upper = 12
+    melee_damage_type = RED_DAMAGE
+    armortype = RED_DAMAGE
+    melee_queue_distance = 3
+    retreat_distance = 2
+    minimum_distance = 1
+    stat_attack = CONSCIOUS
+    attack_verb_continuous = "stomps"
+    attack_verb_simple = "stomp"
+    deathmessage = "crumbles."
+    faction = list("hostile")
+    start_qliphoth = 1
+
+    work_chances = list(
+                        ABNORMALITY_WORK_INSTINCT = list(40, 40, 50, 50, 50),
+                        ABNORMALITY_WORK_INSIGHT = list(40, 40, 50, 50, 50),
+                        ABNORMALITY_WORK_ATTACHMENT = list(70, 70, 80, 80, 90),
+                        ABNORMALITY_WORK_REPRESSION = list(60, 60, 50, 40, 40),
+                        )
+    work_damage_amount = 5
+    work_damage_type = BLACK_DAMAGE
+
+    var/list/counter1 = list() //from FAN, although changed
+    var/list/counter2 = list()
+
+    ego_list = list(
+        /datum/ego_datum/weapon/dwelling,
```

Comment:
Do not forget to include hearth here as well. Also, make sure there are newlines at the end of each file. Just an empty line, really.

Reply:
Same as the other comment